pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -183,9 +189,10 @@ var _ = Describe("Pod Network", func() {
 	TestPodInterfaceIPBinding := func(vm *v1.VirtualMachineInstance, domain *api.Domain) {
 
 		//For Bridge tests
-		mockNetwork.EXPECT().LinkSetName(dummySwap, newPodInterfaceName).Return(nil)
+		mockNetwork.EXPECT().LinkSetName(primaryPodInterfaceBeforeNameChange, newPodInterfaceName).Return(nil)
 		mockNetwork.EXPECT().LinkByName(primaryPodInterfaceName).Return(dummy, nil).Times(2)
```

Comment:
Shouldn't this line be removed?

Reply:
it's used elsewhere

Reply:
Where? IIUC, this line causes the code of `switchPodInterfaceWithDummy` not to be invoked.

Reply:
BTW, I think there is some gomock bug that causes the `Times` option to be ignored. I tried to change here `Times` to 20 and it still passed.
My assumption is that if the mock is an interface on the tested file, the `Times` is ignored.
But it is not relevant to your PR, just may explain the strange behaviour that having -
```
mockNetwork.EXPECT().LinkByName(primaryPodInterfaceName).Return(dummy, nil).Times(2)
mockNetwork.EXPECT().LinkByName(originalPodInterfaceName).Return(primaryPodInterfaceBeforeNameChange, nil)
mockNetwork.EXPECT().LinkByName(newPodInterfaceName).Return(primaryPodInterfaceAfterNameChange, nil)
```
the tests still pass. Although `LinkByName` is invoked max 2 times.